### PR TITLE
Point ERC20/ERC721 connector at deployed contract address

### PR DESCRIPTION
**Pull request by awrichar**
This includes multiple changes to how the `docker-compose.yml` file is handled:

* `docker-compose.yml` has moved back up to the root directory of the stack (instead of being placed in "init" and "runtime" subfolders). A simple migration check will copy it out of "runtime" into the parent folder if needed.
* The `docker-compose.yml` file will now be rewritten after first-time setup is complete. This allows it to pick up on new things such as deployed contracts. It is notated at the top with comments indicating it is a generated file and should not be edited.
* For overriding Docker configuration values, users are now directed to edit a `docker-compose.override.yml` file, as [supported by docker-compose](https://docs.docker.com/compose/extends).

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -34,4 +34,5 @@ require (
 	golang.org/x/sys v0.0.0-20210420205809-ac73e9fd8988 // indirect
 	golang.org/x/text v0.3.4 // indirect
 	gopkg.in/yaml.v2 v2.4.0
```
Comment (hfuss) [inline go.mod:36]: is v2 still needed then?

File: internal/tokens/erc20erc721/erc20_erc721_provider.go [hunk 2]
```diff
@@ -31,6 +32,17 @@ type ERC20ERC721Provider struct {
 	Stack   *types.Stack
 }
 
+type HexAddress string
```
Comment (hfuss) [inline internal/tokens/erc20erc721/erc20_erc721_provider.go:35]: 🧙🏻

File: go.mod (hunk 1 above)
Comment (awrichar) [reply go.mod:36]: I think a bunch of our dependencies pull it in.

File: internal/stacks/stack_manager.go [hunk 3]
```diff
@@ -339,13 +359,26 @@ func (s *StackManager) ensureInitDirectories() error {
...
-func (s *StackManager) writeDockerCompose(workingDir string, compose *docker.DockerComposeConfig) error {
-	bytes, err := yaml.Marshal(compose)
+func (s *StackManager) writeDockerCompose(compose *docker.DockerComposeConfig) error {
+	comments := "# This file is generated - DO NOT EDIT!\n# To override config, edit docker-compose.override.yml\n"
+	bytes := []byte(comments)
+	yamlBytes, err := yaml.Marshal(compose)
 	if err != nil {
 		return err
 	}
+	bytes = append(bytes, yamlBytes...)
```
Comment (nguyer) [inline internal/stacks/stack_manager.go:369]: I guess this works 🙂

**Comment by nguyer** [comment]
Changes look good to me. I'm going to run E2E on these changes in a few variations as a quick smoke test.

File: internal/stacks/stack_manager.go (hunk 3 above)
Comment (awrichar) [reply internal/stacks/stack_manager.go:369]: Yea... wondered if there was a more efficient way?

**Review by nguyer — APPROVED** [review]

**Pull request merged**